src/plugins/music-xml-viewer/music-xml-viewer.less

```diff
@@ -0,0 +1,12 @@
+.MusicXmlViewer {}
+
+.MusicXmlViewer-viewer {
+  margin: 0 auto;
+}
+
+.MusicXmlViewer-caption {
```

Comment:
All these 3 classes could safely be called  `MusicXmlViewerDisplay[-viewer/caption]` as they are only used in the display component.

Reply:
Done (and also done for the PdfViewer[Display], which is almost the same plugin)!

Reply:
Poifect 👍